src/apache/utilities/apache-utilities

```diff
@@ -52,6 +52,21 @@ apache_pid()
 	fi
 }
 
+apache_listen_address()
+{
+    address="$(snapctl get listen-address)"
+    if [ -z $address ]; then
+        address=""
```

Comment:
I'd like to see something more along [these lines](https://github.com/nextcloud/nextcloud-snap/pull/416/files#diff-0b502a9dcd7c3bf1d2d17b3d3ce358d1R74) in this case.